console.log('hello')

add = (x, y) -> x + y

throw new Error('e')
